### ISSUE TITLE
Update filename example shows modified path

### ISSUE THREAD

**Issue by rossheninger** [dylanljones/pyrekordbox#81]
Shouldn't `OSC_SAMPLER/` remain in the path if all you're doing is updating the name?

```
Updating the file name changes the database entry

        >>> db = Rekordbox6Database()
        >>> cont = db.get_content()[0]
        >>> cont.FolderPath
        C:/Music/PioneerDJ/Sampler/OSC_SAMPLER/PRESET ONESHOT/NOISE.wav

        >>> new_name = "noise"
        >>> db.update_content_filename(cont, new_name)
        >>> cont.FolderPath
        C:/Music/PioneerDJ/Sampler/PRESET ONESHOT/noise.wav
```

I assume it was just copy/pasted incorrectly from the update path function.

https://github.com/dylanljones/pyrekordbox/blob/769f1188c26c91dee9defcd17f27569498992a01/pyrekordbox/db6/database.py#L835

**Comment by rossheninger** [comment]
@dylanljones any chance you have a working version of the Rekordbox 6.5.3 installer?

**Comment by dylanljones** [comment]
Yes, good catch! Damn copy paste:D 
I just checked, of course I still have the RB v5 installer, but not the one for 6.5.3...

**Comment by rossheninger** [comment]
Dang, that's a bummer. I'm very ashamed to admit that last night after failing to get this to work with 6.7.something I found a reddit comment you made about the change and decided to back port to 6.5.3. Turns out Rekordbox doesn't host older versions anymore, so I googled around for the installer and like an absolute fool downloaded it from filehorse . com :/

It was late, I was tired and frustrated and rushing lol, everything looked normal until RB startup then it said login failed and I needed to login again, but it looked suspicious, the forgot password link was ad link to some music thing called kuvo...

I immediately panicked as I fully understand how dumb that was as a dev on cloud security service (was on personal computer). I then shut off wifi and bluetooth immediately as to not let any data be sent out in the even of a key logger and was up till 4 am trying to fix my library issues while I had 6.5.3 installed still (the malware seemed to piggyback on the Rekordbox app, maybe a dynamic link library exploit?). 

I was able to use your code well enough, getting the content from the db and filtering out everything I didn't need. The problem I am facing is that I was attempting to do some file organization after 2 years of djing and I had about 4k aiff files, littered with duplicates and inconsistent paths. I wrote a script a few days ago to migrate them all to one folder with a consistent naming schema. Only after doing that did I realize Rekordbox autorelocate only works for path changes not filename changes! Hence the need for your app.

In your library it was functionally doing  
```
db.update_content_filename(content, "content.Artist.Name - str(content.ReleaseYear) - content.Title", True, False)
db.update_content_path(content, "content.Artist.Name - str(content.ReleaseYear) - content.Title", True, True)
```
However that didn't work for some reason I was hitting an error on the filename update when it tried to validate the path of the analysis file. I'm not sure why and I gave up at that point and booted my computer in safe mode and removed the malware. Not sure if you've ever experienced that. I think in my collection I had seen inconsistencies in some files showing that they had analysis files and some not, even though I thought for sure I had analyzed them all. Unfortunately I lost the error message due to reboot so this probably isn't much help.

**Comment by dylanljones** [comment]
Fixed the path, thanks again for pointing that out!
As far as i know, KUVO is by Pioneer and part of Rekordbox. But its always better to stay away from sites like that;)
Please feel free to open a new issue if you run into the error again!

**Comment by rossheninger** [comment]
Realized Kuvo was pioneer after posting that lol, but I wasn't completely paranoid, as file horse had installed a crypto miner or something that was stealing all my cpu.
